internal/mysql/example/queries.sql

```diff
@@ -1,3 +1,4 @@
+-- +goose Up
```

Comment:
This isn't how goose migrations work. There should be a single `Up` and a single `Down` comment per migration. You'll need to remove the second Up and Down comment.